src/gui/executionlogwidget.h

```diff
@@ -30,33 +30,31 @@
 #define EXECUTIONLOGWIDGET_H
 
 #include <QWidget>
+
 #include "base/logger.h"
 
-class LogListWidget;
 
 namespace Ui
 {
     class ExecutionLogWidget;
 }
 
+class LogFilterModel;
+class LogListView;
+
 class ExecutionLogWidget : public QWidget
 {
     Q_OBJECT
 
 public:
-    ExecutionLogWidget(QWidget *parent, const Log::MsgTypes &types);
-    void showMsgTypes(const Log::MsgTypes &types);
+    ExecutionLogWidget(QWidget *parent, Log::MsgTypes types);
+    void showMsgTypes(Log::MsgTypes types);
     ~ExecutionLogWidget();
 
-private slots:
-    void addLogMessage(const Log::Msg &msg);
-    void addPeerMessage(const Log::Peer &peer);
-
 private:
     Ui::ExecutionLogWidget *m_ui;
-
-    LogListWidget *m_msgList;
-    LogListWidget *m_peerList;
+    LogListView *m_msgList, *m_peerList;
```

Comment:
seems this project prefers each variable to have its own line. check other places too!

Reply:
seems like they're not needed to explicitly stored, so I removed them as a member variable.